src/pijama_ast/src/lib.rs

```diff
@@ -129,13 +139,16 @@ impl<'a> Display for Primitive {
     }
 }
 
-/// Encapsulates a conditional where "if `cond` then `body`"
+/// Encapsulates a conditional statement in Pijama's syntax of the 
+/// form "if `cond` then `body`".
 #[derive(Debug, Eq, PartialEq)]
 pub struct Branch<'a> {
     pub cond: Located<Block<'a>>,
     pub body: Located<Block<'a>>,
 }
 
+/// A [`Node`] in the AST that encapsulates the different
+/// expressions and statements that Pijama's syntax supports.
 #[derive(Debug, Eq, PartialEq)]
 pub enum Node<'a> {
     BinaryOp(BinOp, Box<Located<Node<'a>>>, Box<Located<Node<'a>>>),
```

Comment:
Could you add an small doc fragment for each variant here?

Reply:
Once I added `#![deny(missing_docs)]`, I got a whole bunch of lints telling me to do exactly that :joy: